consensus/istanbul/backend/announce.go

```diff
@@ -589,130 +649,311 @@ func (sb *Backend) regossipAnnounce(msg *istanbul.Message, payload []byte) error
 	return nil
```

Comment:
On line 632, because this is the same as the new default announce period, and we do not expect most users to be tweaking that parameter, they are running up against race condition with this DDoS protection. Assuming normally distributed network latency and announce processing time, the message will be rejected 50% of the time when the gossip interval equals the DDoS protection limit. DDoS protection should be adjusted down, or announce interval should be adjusted up. Additional though here is that because these messages range in size so much, we might want to impose a more nuanced limit, like 100 entries or 10MB per 5 minutes. Here is a simply implementation of that idea I was able to whip up https://play.golang.org/p/A0Y9IUCLXRP